File: datagym-backend/src/main/java/ai/datagym/application/labelConfiguration/entity/geometry/LcEntryLine.java
package ai.datagym.application.labelConfiguration.entity.geometry;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;
import java.io.Serializable;

@Entity
@DiscriminatorValue(value = "line")
public class LcEntryLine extends LcEntryGeometry implements Serializable {
    public LcEntryLine() {
        super();
    }
}

File: datagym-frontend/src/app/label-config/model/geometry/LcEntryLine.ts
import { LcEntryGeometry } from './LcEntryGeometry';
import { LcEntryType } from '../LcEntryType';

export class LcEntryLine extends LcEntryGeometry {
  static type: LcEntryType = LcEntryType.LINE;

  constructor(lcEntryParentId: string | null, key?: string, value?: string, color?: string, shortcut?: string) {
    super(LcEntryLine.type, lcEntryParentId, key, value, color, shortcut);
  }
}
